checkFileSize.R

```diff
@@ -0,0 +1,31 @@
+args = commandArgs(trailingOnly=TRUE)
+# test if there is are 2 arguments: if not, return an error and the actual usage
+if (length(args)!=2) {
+          stop("Usage: Rscript check.R <root_folder> <json_file>.\nThe json file should list all the info of the bucket: aws s3api list-objects-v2 --bucket <bucket> --prefix
```

Comment:
The name of the script in the message should be checkFileSize.R instead of check.R.

Reply:
good catch!